gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,3 +1,6 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more contributor
```

Comment:
here also.

Reply:
Scancode does not enforce mini or full against any specific file type... it just cares one of the other is present in the top X lines of the source file.

Reply:
My bad - this was a user error. I've now fixed this as instructed.